tests/testthat/test-loading_workbook.R

```diff
@@ -902,3 +902,10 @@ test_that("Loading readTest.xlsx Sheet 1", {
 
   expect_equal(expected_shared_strings, wb$sharedStrings)
 })
+
+test_that("Loading multiple pivot tables: loadPivotTables.xlsx works",{
+  ## loadPivotTables.xlsx is a file with 3 pivot tables and 2 of them have the same reference data.
+  fl <- system.file("extdata", "loadPivotTables.xlsx", package = "openxlsx")
+  wb <- loadWorkbook(fl)
+  
```

Comment:
FYI the actual test is missing. This should most likely include any kind of `test_` for `wb` e.g., is the data correctly loaded. I'm not even sure if testthat does complain if loadworkbook failed.

Reply:
@JanMarvin yes you are correct. It needs something like `expect_equal()`.

Reply:
@JanMarvin yes you are right, it will be cleaner to have an `expect_` function in the test. I will add it.

For information, if any error occurs inside test_that() function, testthat raises an error.
This test without the code modification creates an error.